packages/react/collapsible/src/Collapsible.tsx

```diff
@@ -184,6 +187,7 @@ const CollapsibleContentImpl = React.forwardRef<
       ref={composedRefs}
       style={{
         [`--radix-collapsible-content-height` as any]: height ? `${height}px` : undefined,
+        [`--radix-collapsible-content-width` as any]: width ? `${width}px` : undefined,
```

Comment:
Since this component is re-composed in `Accordion`, would you mind re-aliasing it there as part of this PR too please?

```jsx
['--radix-accordion-content-width' as any]: 'var(--radix-collapsible-content-width)',
```

Reply:
Sure - makes sense, added a usage of this in stories and the animation does look decent :smile: